tests/_runtime/packages/test_package_managers.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import annotations
+
+from typing import List
```

Comment:
Since we dropped Python 3.8, we can just use `list` now — `typing.List` is deprecated since Python 3.9.

https://docs.python.org/3/library/typing.html#typing.List